Code/PgSQL/rdkit/expected/props.out

```diff
@@ -232,7 +232,8 @@ SELECT mol_numrotatablebonds('CCCC'::mol) mol_numrotatablebonds;
                      1
 (1 row)
 
-SELECT mol_numrotatablebonds('c1ccccc1c1ccc(CCC)cc1'::mol) mol_numrotatablebonds;
+-- mol_from_smiles() shouldn't be necessary, but there's an RDKit bug
```

Comment:
would it make sense to include in the comment the github issue number for the mentioned bug?